index.html

```diff
@@ -11,24 +11,33 @@
     <title>Document</title>
 </head> 
 <body> 
-    <header>
-        <ul>
-            <li><img src="images/Logo.png" alt="logo"></li>
-            <li><i class="fa-solid fa-bars"></i></li>
-        </ul>
-    </header>
-    <section class="Headline"> 
-        <h1>I’m kelvin Glad to see you!</h1>
-        <p id="one">I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-        <p id="two">LET'S CONNECT</p>
-        <a href="#"></a>
-        <ul> 
-            <li><a class="fa-brands fa-twitter" href="https://twitter.com/kevin_ngotho?s=09"></a></li>
-            <li><a class="fa-brands fa-linkedin" href="https://www.linkedin.com/in/kelvin-kaviku-5178001a6"></a></li>
-            <li><a class="fa-brands fa-medium" href="#"></a></li>
-            <li><a class="fa-brands fa-github" href="https://github.com/brainskev"></a></li>
-            <li><a class="fa-solid fa-hand-peace" href="#"></a></li>
-        </ul>
-    </section>
+    <div class="flex-container">
+        <div style="align-self: left"></div>
+            <header>
+                <ul>
+                    <li><img src="images/menu.png" alt="logo"></li>
+                    <!--<li><i class="fa-solid fa-bars"></i></li>-->
+                </ul>
+            </header>
+        </div>
+        <div>
+            <section class="Headline"> 
+                <h1>I’m kelvin Glad to see you!</h1>
+                <p id="one">I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <p id="two">LET'S CONNECT</p>
+                <a href="#"></a>
+                <ul> 
+                    <li><a class="fa-brands fa-twitter" href="https://twitter.com/kevin_ngotho?s=09"></a></li>
+                    <li><a class="fa-brands fa-linkedin" href="https://www.linkedin.com/in/kelvin-kaviku-5178001a6"></a></li>
+                    <li><a class="fa-brands fa-medium" href="#"></a></li>
+                    <li><a class="fa-brands fa-github" href="https://github.com/brainskev"></a></li>
+                    <li><a class="fa-solid fa-hand-peace" href="#"></a></li>
+                </ul>
```

Comment:
- [ ] It's a good practice to follow the design template provided closely in terms of margins, colours etc. The icons here need a bit more styling. Kindly fix that.
- [ ] I also recommend that you adjust the spacing between the elements to match that of the template.

Figma Design  | Your Design
--|--
![image](https://user-images.githubusercontent.com/29071984/198314386-fc04d04f-7b55-4ef7-8cbc-9e5be1c5b4e3.png) | ![image](https://user-images.githubusercontent.com/29071984/198315157-50241f34-1836-40ac-804f-a9869587d6dd.png)